packages/strapi-parts/src/DatePicker/DatePicker.stories.mdx

```diff
@@ -12,7 +13,21 @@ This is the doc of the `DatePicker` component
 
 ## DatePicker
 
-Description...
+Date pickers help users selecting a specific date from a calendar. They can be associated to a Time picker component.  
+
+**Best practices**
+
+- When the date picker is clicked, it takes today's date.
+- It is not possible to write the date from the keyboard.
+- It is not possible to change the date format. However, the date format will be based on the browser locale. 
+- A Date picker can used simultaneously with a Time picker in a form.
```

Comment:
I'm not sure this is available ? @mfrachet

Reply:
This has not been spec-ed, and thus, not been implemented. For now, if you want to use both, you have to use a DatePicker and a TimePicker

Reply:
We'll have to then, find a way to achieve that as it's part of the fields possibilities in the CTB :) Happy to discuss it